draft-sharabayko-mops-srt.md

```diff
@@ -1249,6 +1239,25 @@ of the handshake Extension Message Flags to 1.
 In this mode consecutive packets form one continuous stream that can be read, with 
 portions of any size.
 
+### Use Case Live Streaming {#use-case-live-streaming}
+
+This section describes real world examples of live audio/video streaming and the current consensus, which is maintaining compatibility between SRT implementations by different vendors. It's meant as guidance for developers to be compatible with existing SRT implementations.
+
+The term "live streaming" refers to MPEG-TS style continuous data transmission with latency management. Live streaming based on segmentation and transmission of files like in HLS protocol is not part of this use case.
+
+The default mode for continuous live streaming is Message Mode. 
+
+Only data packets with their PP field set to "11b" are allowed, meaning a single data packet fits one message.
+
+Additionally Timestamp-Based Packet Delivery (TSBPD) ({{tsbpd}}) and
+Too-Late Packet Drop ({{too-late-packet-drop}}) mechanisms are used in this mode.
+
+The Order Flag ({{data-pkt}}) needs special attention. In the case of live streaming it is set to 0, which would allow out of order packet delivery. However, in this use case the Order Flag has to be ignored by the receiver. As Timestamp-Based Packet Delivery (TSBPD) is enabled, the receiver will still deliver packets in order, but based on timestamps. An exception would be packets arriving too late and are skipped by Too-Late Packet Drop mechanism. (See {{too-late-packet-drop}})
```

Comment:
> An exception would be packets arriving too late and are skipped by Too-Late Packet Drop mechanism.

In the case of a packet arriving too late and skipped by Too-Late Packet Drop mechanism, the order of delivery is still maintained except for potential sequence discontinuity.